content-src/components/DiscoveryStreamComponents/DSCard/DSCard.jsx

```diff
@@ -78,26 +79,28 @@ export class DSCard extends React.PureComponent {
 
   render() {
     return (
-      <a href={this.props.url} className="ds-card" onClick={this.onLinkClick} ref={this.cardElementRef}>
-        <div className="img-wrapper">
-          <div className="img" style={{backgroundImage: `url(${this.props.image_src}`}} />
-        </div>
-        <div className="meta">
-          <div className="info-wrap">
-            <header className="title">{this.props.title}</header>
-            {this.props.excerpt && <p className="excerpt">{this.props.excerpt}</p>}
+      <div ref={this.cardElementRef}>
```

Comment:
keeping the ref prop out of SafeAnchor as we are overriding this logic in bug 1519879 where we will pull out logic that handles intersection observer in DSCard as a separate component